<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightPanel</class>
 <widget class="QWidget" name="RightPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPushButton" name="plotButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add a new plot</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/icons/addPlot.png) 0 0 0 0 stretch stretch;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="actionsButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>55</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add a new action</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/icons/actions-list.png) 0 0 0 0 stretch stretch;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="connectionButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Start a connection</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/icons/add_con.png) 0 0 0 0 stretch stretch;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="stimulationButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Stimulate this node</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/icons/injection.png) 0 0 0 0 stretch stretch;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="stimulationFrame">
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>450</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#aFrame,#bFrame,#cFrame{
background: rgba(100,100,100,100);
}
#outFrame{
background: rgba(210,210,210,220);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="outFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QFrame" name="aFrame">
           <property name="minimumSize">
            <size>
             <width>303</width>
             <height>169</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>393</width>
             <height>169</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QSlider" name="stimulusSlider">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>4000</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="maxMinLabel">
                  <property name="text">
                   <string>Max:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="maxSpinBox">
                  <property name="toolTip">
                   <string>Maximum absolute value of stimulus slider.</string>
                  </property>
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                  <property name="singleStep">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>100000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="negativeCheckBox">
                  <property name="text">
                   <string>neg. values</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QRadioButton" name="stimulusFullButton">
                <property name="text">
                 <string>Single value</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QRadioButton" name="stimulusSomeButton">
                <property name="text">
                 <string>From 0 to 10</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="stimulusLabel_6">
                  <property name="text">
                   <string>      First:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="startSpinBox"/>
                </item>
                <item>
                 <widget class="QLabel" name="stimulusLabel_5">
                  <property name="text">
                   <string>Size:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sizeSpinBox"/>
                </item>
               </layout>
              </item>
              <item row="6" column="0">
               <widget class="QSlider" name="stimulusSomeSlider">
                <property name="maximum">
                 <number>359</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="stimLabel">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Current injection (I=0)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="bFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QCheckBox" name="oscCheckBox">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Oscillation (No freq)</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="1">
               <widget class="QSlider" name="oscFrequencySlider">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Frequency:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Phase:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Amplitude:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="oscPhaseSpinBox">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>6.283200000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSlider" name="oscAmpSlider">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="cFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QCustomPlot" name="plot" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
        <zorder>bFrame</zorder>
        <zorder>aFrame</zorder>
        <zorder>verticalSpacer</zorder>
        <zorder>cFrame</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="actionFrame">
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#actionFrameA{
background: rgba(100,100,100,100);
}
#actionFrameOut{
background: rgba(210,210,210,220);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QFrame" name="actionFrameA">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>parameter:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>value:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="actionParameter">
             <item>
              <property name="text">
               <string>sync</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>learn</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="actionValue"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="actionExecuteButton">
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>source/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
